js/shipment.js

```diff
@@ -95,6 +103,15 @@ export const ShipmentAPI = superclass =>
             return response;
         }
 
+        /**
+         * Uploads a document to the DHL server for a shipment.
+         *
+         * @param {String} trackingNumber The tracking number of the shipment/waybill
+         * @param {Object} payload The payload object according to the DHL API standards.
+         * @param {Object} options An object of options to configure the request.
+         * @returns {Object} The HTTP response object.
+         * @see https://developer.dhl.com/api-reference/dhl-express-mydhl-api#reference-docs-section/operations-pickup-exp-api-pickups

```

Comment:
Let's use "https://developer.dhl.com/api-reference/dhl-express-mydhl-api#reference-docs-section". Do this in the rest of the links